src/App.tsx

```diff
@@ -1,61 +1,27 @@
 import './App.scss';
+import { useState, FC } from 'react';
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search. In your case, you're using the `FC` alias for `FunctionComponent`. It would be better to use the full name for clarity.